src/api/uploads/__tests__/FolderUploadNode.test.js

```diff
@@ -116,6 +116,26 @@ describe('api/uploads/FolderUploadNode', () => {
             expect(folderUploadNodeInstance.folderId).toBe(folderId);
         });
 
+        test('should not set error for non-root folders if error is ITEM_NAME_IN_USE', async () => {
+            const folderId = '1';
+            const errorCallback = jest.fn();
+            const isRoot = false;
+            const error = {
+                code: ERROR_CODE_ITEM_NAME_IN_USE,
+                context_info: { conflicts: [{ id: folderId }] },
+            };
+            folderUploadNodeInstance.name = name;
+            folderUploadNodeInstance.createFolder = jest.fn(() => ({
```

Comment:
this prevents the test from hitting the catch block in the code. it should use a promise that either resolves or rejects. In this case it should reject with `error` to mock the code being tested.